packages/xpetra/src/Import/Xpetra_EpetraImport.cpp

```diff
@@ -114,8 +119,8 @@ template RCP<const Import<int,long long,Kokkos::Compat::KokkosThreadsWrapperNode
 template class EpetraImportT<long long, Kokkos::Compat::KokkosOpenMPWrapperNode >;
 template RCP<const Import<int, long long, Kokkos::Compat::KokkosOpenMPWrapperNode> > toXpetra<long long,Kokkos::Compat::KokkosOpenMPWrapperNode>(const Epetra_Import *);
 #endif
-#ifdef HAVE_TPETRA_INST_CUDA
-typedef Kokkos::Compat::KokkosCudaWrapperNode default_node_type;
+#ifdef HAVE_TPETRA_INST_HIP
```

Comment:
Why was the HAVE_TPETRA_INST_CUDA clause removed?

Reply:
Fixed